virtcontainers/persist/api/interface.go

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2018 Huawei Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package persistapi
+
+// PersistDriver is interface describing operations to save/restore persist data
+type PersistDriver interface {
+	// ToDisk flushes data to disk(or other storage media such as a remote db)
+	ToDisk() error
```

Comment:
This is a good descriptive name, but -- in my mind atleast -- this needs to be paired with a  `FromDisk()` call (rather than `Restore()`).

But maybe you could just rename this to `Save()` (or `Persist()`) as that would fit naturally with `Restore()`.

Reply:
`FromDisk()` sounds good.

Also "Load()" from `store`  package  sounds acceptable to me too.